Automattic/newspack-plugin
# Advertising Wizard with AdSense

### All Submissions:


### Changes proposed in this Pull Request:

Implements the initial advertising wizard, with AdSense setup as the first option. This just redirects to the AdSense setup screen within the Site Kit plugin right now.

### How to test the changes in this Pull Request:

1. Apply the patch
2. Run `npm ci && npm run clean && npm run build:webpack`
3. Observe the new "Advertising" element on the main Newspack dashboard
4. Observe the Checklist after clicking on "Advertising"
5. Check the plugin installer presents itself if the Site Kit plugin isn't installed/activate
6. Check the last step presents the "Manage Site Kit" buttons
7. Check the button takes you to the AdSense module of Site Kit

### Other information:
